### ISSUE TITLE
Fix Google Search Console's Complaint About Sitemap Index Nesting

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#30]
# Description
```
Please provide a short description of what you have encountered below.
```
It seems like Google Search Console disallows Index type sitemap nesting. Self-discovery is likely not possible until they opens up and permits it.

Some brainstorm was:

1. Maintain `sitemap-page.xml` - unify all the pages' content URLs (it's a multiplier that we have to get rid).
2. Only generate 1 `sitemap.xml` (index) file at root level - requires recursive scanning.

# Expected Behavior
```
Please specify the expected behavior of your usage below.
```
`sitemap.xml` works like industrial application.

# Current Behavior
```
Please specify the current behavior of software below.
```
GSC rejected the index sitemap nesting work.

# Steps to Reproduce (COMPULSORY)
```
Please specify all the steps required to re-produce the problem.
```

# Attachments
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by hollowaykeanho** [comment]
Google Search Console feedback submitted.

**Comment by hollowaykeanho** [comment]
Not possbile. If we scan all pages even using 1 page, it will head back to Issue #26 where current compilation takes from 20s to >5 mins. 

This is far more unusable than before.

**Comment by hollowaykeanho** [comment]
Managed to develop a way to scan all pages only at root page while maintaining performance (commit: https://github.com/ZORALab/Hestia/commit/7a6003c5b09a8e6e06aa557c261a357590881979). 

[metric-stage-5.ods](https://github.com/ZORALab/Hestia/files/10299409/metric-stage-X02.ods)
[metric-stage-5.txt](https://github.com/ZORALab/Hestia/files/10299412/metric-stage-5.txt)
[metric-stage-4.ods](https://github.com/ZORALab/Hestia/files/10299411/metric-stage-4.ods)
[metric-stage-4.txt](https://github.com/ZORALab/Hestia/files/10299413/metric-stage-4.txt)

Begin reconstruct root `sitemap.xml` for all localized index files and repository level index file.

**Comment by hollowaykeanho** [comment]
Fixed in 04a3110b073e67d2c68634263f7ac8e0b1e23f77

**Comment by hollowaykeanho** [comment]
Released in https://github.com/ZORALab/Hestia/releases/tag/v1.0.0
